sqlx-core/src/pool/mod.rs

```diff
@@ -141,6 +142,20 @@ where
     }
 }
 
+impl<DB: Database> fmt::Debug for Pool<DB> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "Pool {{ url: {}, size: {}, num_idle: {}, closed: {}, options: {:?} }}",
+            self.inner.url(),
+            self.inner.size(),
+            self.inner.num_idle(),
+            self.inner.closed(),
+            self.inner.options()
+        )
```

Comment:
I was thinking something more like this: https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html#examples

Reply:
fixed

Reply:
Excellent!